Src/Plugins/Security/Security.csproj

```diff
@@ -24,12 +24,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="GoogleApi" Version="4.0.4" />
+    <PackageReference Include="BouncyCastle.Cryptography" Version="2.2.1" />
```

Comment:
>BouncyCastle [](http://example.com/codeflow?start=31&length=12)

why do we need this?

Reply:
`BouncyCastle.Cryptography` is used by [`CryptographicPrivateKeyValidator`](https://github.com/microsoft/sarif-pattern-matcher/blob/7c5683e62f5eeb3e27b64a0227d58cd9934c0c20/Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_013.CryptographicPrivateKeyValidator.cs#L14)

It is included in `GoogleApi` package, so need to add it back.

Reply:
https://github.com/microsoft/sarif-pattern-matcher/blob/7c5683e62f5eeb3e27b64a0227d58cd9934c0c20/Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_013.CryptographicPrivateKeyValidator.cs#L14